apps/aehttp/test/aehttp_ws_test_utils.erl

```diff
@@ -0,0 +1,155 @@
+-module(aehttp_ws_test_utils).
+-behaviour(websocket_client).
+
+%% API
+-export([start/2,
+         send/4,
+         subscribe_for_event/3,
+         unsubscribe_for_event/3,
+         wait_for_event/2,
+         wait_for_event/3,
+         stop/1]).
+
+%% behaviour exports
+-export([
+         init/1,
+         onconnect/2,
+         ondisconnect/2,
+         websocket_handle/3,
+         websocket_info/3,
+         websocket_terminate/3
+        ]).
+
+start(Host, Port) ->
+    crypto:start(),
+    ssl:start(),
```

Comment:
I see these are application:start of crypto and ssl. Shall these two lines be removed from here and moved to aehttp_integration_SUITE:init_per_group websocket? (And maybe corresponding stops in end_per_group.)
Please consider (I am not sure what current design in tests is):
* Matching return value (so to fail at correct point if e.g. crypto cannot start);
* Using start/ensure_started/ensure_all_started and stop.

BTW I see these crypto an ssl are [dependencies of websocket_client](https://github.com/sanmiguel/websocket_client/blob/2b6228211389fbd0b1694e5b71e33da48d98d2b7/src/websocket_client.app.src#L5)

Reply:
I see - originally taken from https://github.com/sanmiguel/websocket_client/tree/2b6228211389fbd0b1694e5b71e33da48d98d2b7#usage